content/docs/dvclive/api-reference/log.md

```diff
@@ -0,0 +1,55 @@
+# dvclive.log()
+
+Generates _metrics logs_ (usable by `dvc plots`) by saving the given `name`:
+`val` pair to a `.tsv` file.
+
+```py
+ def log(name: str, val: float, step: int = None):
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+dvclive.log("loss", 0.9)
+```
+
+## Description
+
+The first call to `dvclive.log(name, val)` will create a new file in
+`{path}/{name}.tsv` including the header and first row.
```

Comment:
What is `path` though? Hasn't been explained in this doc. Maybe just link to `init` earlier and avoid this if it's not central to `dvclive.log`